packages/cli/__tests__/main.unit.test.ts

```diff
@@ -29,7 +29,7 @@ describe("behavior of main.ts", () => {
 
         it("should exit with non zero return code if Imperative.init() fails - calling main.ts directly", async () => {
 
-            const mainTs = resolve(__dirname, '../src/main');
+            const mainTs = resolve(__dirname, '../../packages/main');
```

Comment:
The path on the left (`../src/main`) seems like the correct one for monorepo structure

Reply:
Fixed. 

A funny observation, in this view, the diff is above & below, not left & right.  I was so confused by "on the left":
![image](https://user-images.githubusercontent.com/14940561/104109757-f85bac80-529e-11eb-9eab-26febcc2eb31.png)

Reply:
Sorry haha - will try to keep that in mind 🙂